src/components/ZeroLogon/Zerologon.tsx

```diff
@@ -42,16 +42,16 @@ const ZeroLogon = () => {
         "session key.\n\nFurther information can be found at: https://www.crowdstrike.com/blog/cve-2020-1472-zerologon-" +
         "security-advisory/\n\n" +
         "Using ZeroLogon:\n\n" +
-        "Step 1: Enter a Domain Controller name.\n" +
+        "Step 1: Enter a domain Controller name.\n" +
```

Comment:
Lowercase c for controller. 
```suggestion
        "Step 1: Enter a domain controller name.\n" +
```